alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM dkubb/alpine-rails-nginx
+
+# Remove booting the rails app
+RUN rm -r /etc/sv/rails /etc/service/rails
+
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+MAINTAINER Markus Schirp <mbj@schirp-dso.com>
+
+ENV BUILDKITE_AGENT_VERSION=edge                                \
+    BUILDKITE_BUILD_PATH=/opt/buildkite/builds                  \
+    BUILDKITE_HOOKS_PATH=/opt/buildkite/hooks                   \
+    BUILDKITE_BOOTSTRAP_SCRIPT_PATH=/opt/buildkite/bootstrap.sh
```

Comment:
OT: IMO we should create a `buildkite` alpine linux package to get a rid of this altogether. I'm not even sure this should be globally exported. Its likely enough we only export it for the deamon.